bindings/wasm/vfs.js

```diff
@@ -1,32 +1,57 @@
-const fs = require('node:fs');
-
 class VFS {
   constructor() {
+    this.files = new Map();
+    this.nextFd = 1;
   }
 
   open(path, flags) {
-    return fs.openSync(path, flags);
+    const fileHandle = {
```

Comment:
I thought this PR was adding support for OPFS, not replacing Node.js support with it